client/app/components/routes/select-operator/form/UserOperator.tsx

```diff
@@ -107,11 +98,8 @@ export default async function UserOperator({
     ...userOperatorData,
   };
 
-  return params?.id === "request-access" ? (
-    // If the operator exists then show the form from the second page
-    <UserOperatorContactForm formData={formData} schema={userOperatorPage2} />
-  ) : (
-    <UserOperatorMultiStepForm
```

Comment:
If we need to re-add the senior officer form later, I think we should be able to conditionally change the `UserOperatorForm` schema rather conditionally using a different form

Reply:
Yeah we could do that though the design showed a single page form. I was actually going to suggest removing the [formSection] route since I noticed we are still passing the form page. Though I think we can keep it in for now if there is a change we will use the multistep form again.

Reply:
Yes, I don't want to do any drastic refactoring until this is more settled.